src/components/CustomRTE/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function CustomRTE({
   label,
   ...props
 }: CustomRTEProps) {
-  const colorScheme = useColorScheme();
```

Comment:
From `Mantine` document, `useColorScheme` hook uses `prefers-color-scheme` media under the hood.

```ts
export function useColorScheme(initialValue?: 'dark' | 'light') {
  return useMediaQuery('(prefers-color-scheme: dark)', initialValue === 'dark') ? 'dark' : 'light';
}
```

This css media will detect color scheme from user operating system appearance. Unexpectedly, we do not allow user get theme automatically by os setting.

For better understanding. You can test it by changing your os theme from this [matine page](https://mantine.dev/hooks/use-color-scheme/) to see the value change.